api/v1/tetragon/tetragon.proto

```diff
@@ -451,6 +462,35 @@ message GetHealthStatusRequest {
     repeated HealthStatusType event_set = 1;
 }
 
+// Tainted bits to indicate if the kernel was tainted. For further details: https://docs.kernel.org/admin-guide/tainted-kernels.html
+enum TaintedBitsType {
+	TAINT_UNSET = 0;
+
+	/* A proprietary module was loaded. */
+	TaintProprietaryModule = 1;
+
+	/* A module was force loaded. */
+	TaintForcedModule = 2;
+
+	/* A module was force unloaded. */
+	TaintForcedUnloadModule = 4;
+
+	/* A staging driver was loaded. */
+	TaintStagedModule = 1024;
+
+	/* An out of tree module was loaded. */
+	TaintOutOfTreeModule = 4096;
+
+	/* An unsigned module was loaded. Supported only on kernels built with CONFIG_MODULE_SIG option. */
+	TaintUnsignedModule = 8192;
+
+	/* The kernel has been live patched. */
+	TaintKernelLivePatchModule = 32768;
+
+	/* Loading a test module. */
+	TaintTestModule = 262144;
```

Comment:
All our other enum values are UPPER_SNAKE_CASE. Could you change these to match?

Reply:
Alright I did, but we have already merged code in the past that didn't follow this format like SecureBits... is this some proto buf standards?

As long as it is printed as strings in Json I think both format are fine... developers can accommodat